src/System.Management.Automation/engine/Interop/Windows/GetTokenInformation.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1306:FieldNamesMustBeginWithLowerCaseLetter", Justification = "Keep native struct names.")]
+
+    internal static unsafe partial class Windows
+    {
+        internal enum TOKEN_INFORMATION_CLASS
+        {
+            TokenUser = 1,
+            TokenGroups,
+            TokenPrivileges,
+            TokenOwner,
+            TokenPrimaryGroup,
+            TokenDefaultDacl,
+            TokenSource,
+            TokenType,
+            TokenImpersonationLevel,
+            TokenStatistics,
+            TokenRestrictedSids,
+            TokenSessionId,
+            TokenGroupsAndPrivileges,
+            TokenSessionReference,
+            TokenSandBoxInert,
+            TokenAuditPolicy,
+            TokenOrigin
+        }
+
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        internal struct SID_AND_ATTRIBUTES
+        {
+            internal nint Sid;
+            internal uint Attributes;
+        }
+
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        internal struct TOKEN_USER
+        {
+            internal SID_AND_ATTRIBUTES User;
+        }
+
+        [LibraryImport("api-ms-win-security-base-l1-1-0.dll", SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static partial bool GetTokenInformation(
+            nint TokenHandle,
+            TOKEN_INFORMATION_CLASS TokenInformationClass,
+            nint TokenInformation,
```

Comment:
This parameter is declared as `LPVOID` in the API. We should declare as `void*` and avoid the cast at the call site.

Reply:
Replaced with `byte*`.